dataloader.properties

```diff
@@ -97,6 +97,7 @@ loginUrl=https://rest.bullhornstaffing.com/rest-services/login
 #placementChangeRequestExistField=customText1
 #placementCommissionExistField=migrateGUID
 #placementExistField=customText1
+#placementShiftSetExistField=externalID,placement.customText1
```

Comment:
This should probably only be externalID, to keep things simple. There should be no need for a multi-field lookup.